src/news/2017-08-30-self-assessment-toolkit.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Self-Assessment Learning Toolkit
+filename: self-assessment-toolkit
+date: '2017-08-30'
+---
+
+We have been working on the design and implementation of an inclusive self-assessment
+toolkit that will engage learners themselves – supported by personalized learning
+analytics – as aspiring research scientists in the important subject of self-regulation
+and self-determination. The toolkit will engage a broad spectrum of learners in a
+self-directed, personalized learning process, by enabling learners to set flexible
+goals and customizable milestones towards achieving those goals based on their unique needs and preferences.
+
+Recently, the design team has been considering alternate entry points into the toolkit,
+from highly specific to broad and exploratory. Part of this work has been the exploration
+of new ways of engaging the learner through customizable “touchnotes”, which would afford
+the learner a quick and easy manual data collection process. A single data point is
+collected when a touchnote is selected. This can be combined with inputs from automated
+data collection sources (e.g. from a weather app or a personal health app, etc.),
+resulting in a fully personalizable data analysis and self-assessment.
+
+The following images show examples of the designs we are currently developing
+(select to enlarge):
+
+<figure>
+<a href="images/Self-reflection-Figure-1.png"><img src="images/Self-reflection-Figure-1
```

Comment:
None of the images in this article are showing, please fix them.

Reply:
They are definitely not working for me. When I try to load the image on its own, I get the following error in my browser's console:
`GET http://localhost:8080/news/images/Self-reflection-Figure-1-thumb.png net::ERR_INVALID_CHUNKED_ENCODING 200 (OK)`

I believe this is happening because the URLs for the images in the markdown have dashes between each word (`Self-reflection-Figure-1-thumb.png`), but the files in the `dist\news\images` directory have spaces in their names (`Self-reflection Figure 1 thumb.png`)

Reply:
fixed